stable/kube-state-metrics/templates/podsecuritypolicy.yaml

```
@@ -16,6 +16,9 @@ spec:
   privileged: false
   volumes:
     - 'secret'
+{{- if .Values.podSecurityPolicy.additionalVolumes }}
```

Let's change it to having a `volumes parameter` and we can default it to having `secret` in the `values.yaml`
```suggestion
{{- if .Values.podSecurityPolicy.volumes }}
```

---

The list should be empty here.

---

We would lose backward compatibility if default it to empty

---

We default to `['secret']` in `values.yml`, so the default is effectively the same afaik.